src/main/public/js/analytics-custom-event-sender.js

```diff
@@ -13,6 +13,15 @@ $(function () {
     ga('send', 'event', eventCategory, eventAction, eventLabel)
   }
 
+  function escape (value) {
+    if (!value) {
```

Comment:
We should also check that the value is a string.

Reply:
Done.